src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -646,12 +667,12 @@ module Iostats_value = struct
         avgqu_sz = avgqu_sz /. 5.;
       }
 
-  let sumup (values : t list) : t =
+  let accumulate (values : t list) : t =
     List.fold_left (fun acc v ->
```

Comment:
I might have written it this way. I'm not sure accumulate is the best name as this is a point-wise max function.
```
let accumulate (values : t list) : t =
  let max acc v = {
        latency = Float.max acc.latency v.latency;
        avgqu_sz = Float.max acc.avgqu_sz v.avgqu_sz;
      }
  in List.fold_left max empty values
```